data/human/human missions.txt

```diff
@@ -4965,3 +4965,164 @@ mission "Riley"
+			label enter
+			`    You enter the halway behind the opening. The walls are perfectly smooth and the light seems to come out of nowhere. "To use alien technology for something like this," Sophie laughs, "That is totally our dad." You proceed until you enter dome like room. On the walls all kind of alien artifacts line up next to each other. In the center of the room is a small slope with a strange looking sword inside. A small tablet stands on a socket in front of it.`
+			choice
+				"[get closer to the tablet]"
+					goto sword-scene
+				"[take a look around first]"
+			`    You walk around, looking at the various alien artifacts.`
+			`    A lot of the artifacts look similar to Sheragi technology, you ask yourself where he got this stuff.`
+				to display
+					has "Sheragi Archaeology: The Emerald Sword 3: done"
+			`    After some time Sophie calls you both to the tablet.`
+			label sword-scene
+			scene "scene/sword dome"
+			`    You all get closer to the tablet. A short message is written on it:`
+			`    My dear children, this is the summary of all the things I have done, enjoy it!`
+			choice
+				"That is one of the best gifts I have ever seen."
+				"Now this is some cool stuff."
+			`    You leave the two of them alone for a bit, walking around and looking at the atrifacts.`
+			`    After some time they wink you to them. Sophie starts, "Captain <last>, we have thought about this sometime and we think it would be a shame to just keep this sword as a memory. It needs to be used. It was our fathers one and only weapon, which probably means that is has all kind of alien tecnology integrated," she takes the sword and reaches it to you. "It is yours now." You look at her, surprised.`
+			choice
+				"This is too much for me to take."
+					goto decline-offer
+				"I would be more than grateful to take it."
+					goto accept-sword
+			label decline-offer
+			`    "Are you sure," Liam asks, "this is a one time offer."`
+			choice
+				"Yes, I am sure."
+				"Well, if you say it like this, I will take it."
+					goto accept-sword
+			`    "I see, at least take this extra payment," Liam transfers 50 thousand credits to your bank account.`
+			action
+				payment 50000
+			`    "It is for you traveling so much with us," Sophie adds.`
+				goto end
+			label accept-sword
+			`    "Please take it on some more adventures," Sophie says. You take the sword. It feels strangely light and as you swing it an energy bolts shoots out of it. Better be careful with his.`
+			action
+				outfit "Old Man's sword" 1
+			label end
+			`    "Well, it seems like this is the end of our short journey together, Capatin <last>," Sophie says. "It was a pleasure to travel with you," says Liam. You say goodbye to them and leave them alone in the dome.`
+		payment 50000
```

Comment:
```suggestion
			label accept-sword
			`	"Please take it on some more adventures," Sophie says.`
			action
				outfit "Old Man's Sword" 1
			label end
			`	"Well, it seems like this is the end of our short journey together, Captain <last>," Sophie says.`
			`	"It was a pleasure to travel with you," adds Liam. "We will always thank you for helping us with our father's will. Here's your payment of <payment>, and farewell." `
			`	You say goodbye to them and leave them alone in the dome.`
				payment 50000
				decline
```
I think the part about the energy bolts should be 'learned' after using it, you did say that you wanted a part of it to be unknown. I also think that a potential 100,000 credits for this mission is massive, although you do bring them around the entirety of human space.

Reply:
If you think about this, the amount of jumps needed, equals even more than 100,000 but I am open to changes